dune-project

```diff
@@ -0,0 +1 @@
+(lang dune 3.12)
```

Comment:
This should be moved down, since we currently make each example an independent dune project.